docs/core/diagnostics/compare-metric-apis.md

```diff
@@ -68,10 +68,8 @@ access to the aggregated values, and has limitations when using more than one li
 [dotnet-counters](dotnet-counters.md), and [dotnet-monitor](https://devblogs.microsoft.com/dotnet/introducing-dotnet-monitor/). For third-party
 tool support, check the vendor or project documentation to see if it's available.
 
-At the time of writing, this is the cross-platform .NET runtime API that has the broadest and most stable ecosystem support. However, it will likely be
-overtaken soon by growing support for [System.Diagnostics.Metrics](metrics-instrumentation.md). The .NET team doesn't expect to
-make substantial new investments on this API going forward, but as with `PerformanceCounters`, the API remains actively supported for all
-current and future users.
+The .NET team doesn't expect to make substantial new investments on this API going forward, but as with `PerformanceCounters`, the API remains
+actively supported for all current and future users.
```

Comment:
Should we write something like `these APIs may get obseleted in the future. Users moving forward should consider using Metrics APIs?

Reply:
I'm fine adding stronger messaging to encourage folks towards Metrics but I think its premature to be suggesting obsoletion. At this point not all the tools that support EventCounters also support S.D.Metrics and we have no built-in runtime or process metrics.

Reply:
For HTTP, semantics of EventCounters and their metrics counterparts are not equivalent since we had to adapt OTel recommendations with the latter. In some cases the old EventCounter semantics can be more useful, meaning that obsoleting them would require us to implement additional instruments that mirror the old semantics.